internal/xds/translator/utils.go

```diff
@@ -66,6 +67,7 @@ func url2Cluster(strURL string) (*urlCluster, error) {
 		hostname:     u.Hostname(),
 		port:         uint32(port),
```

Comment:
we also need to have a default port for `http`, otherwise the user is required to explicitly mention the port when `http` is being used, in this case, `80`.